src/inmanta/agent/handler.py

```diff
@@ -354,16 +395,17 @@ def update_changes(  # noqa: F811
     def changes(self) -> Dict[str, AttributeStateChange]:
         return self._changes
 
-    def log_msg(self, level: int, msg: str, args: Sequence[object], kwargs: Dict[str, object]) -> None:
+    def log_msg(self, level: int, msg: str, args: abc.Sequence[object], kwargs: abc.MutableMapping[str, object]) -> None:
+        LOGGER.warning("Direct calls to the log_msg method are being deprecated, please use the LoggerABC interface instead.")
+        self._log_msg(level, msg, *args, **kwargs)
```

Comment:
mypy now complains on this because its expecting arg4 to be a bool, is there a way around this ? This would go into iso6 only in the end

Reply:
I'd ignore it. It's the least of our typing errors and as you say it's not present on master. We could circumvent it but it would lead us back to more or less what you had before and in my opinion this is the cleaner approach.